website/source/docs/commands/snapshot/agent.html.markdown.erb

```diff
@@ -205,6 +207,49 @@ Note that despite the AWS references, any S3-compatible endpoint can be specifie
 
 * `-aws-s3-kms-key` -  Optional Amazon KMS key to use, if this is not set the default KMS master key will be used. Set this if you want to manage key rotation yourself.
 
+#### S3 Required Permissions
+
+Different S3 permissions are required depending on the configuration of the snapshot agent. In particular extra permissions are required when
+snapshot rotation is enabled. S3 storage snapshot rotation is enabled when the `retain` configuration is greater than 0 and when there is
+no `aws-s3-static-snapshot-name` configured.
+
+| Permission           | Resource                           | When you need it                                |
+| -------------------- | ---------------------------------- | ----------------------------------------------- |
+| `PutObject`          | `arn:aws:s3:::<bucket name>/<key>` | Required for all operations.                    |
+| `DeleteObject`       | `arn:aws:s3:::<bucket name>/<key>` | Required only when snapshot rotation is enabled |
+| `ListBucket`         | `arn:aws:s3:::<bucket name>`       | Required only when snapshot rotation is enabled |
+| `ListBucketVersions` | `arn:aws:s3:::<bucket name>`       | Required only when snapshot rotation is enabled |
+
+Within the table `<key>` refers to the the key used to store the snapshot. When `aws-s3-static-snapshot-name` is configured the `<key>` is simply the value of that configuration. Otherwise the `<key>` will be the `<aws-s3-key-prefix configuration>/consul-*.snap`.
+
+The following example IAM policy document assumes that the `aws-s3-bucket` is `consul-data` with defaults for `aws-s3-key-prefix`, `aws-s3-static-snapshot-name` and `retain`
```

Comment:
Minor but probably some kind of punctuation at the end of this line (colon, period).